### ISSUE TITLE
Create new Boutiques support modules InputValueFixer

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#1188]
Using the new support for modules in the new Boutiques integrator (#1181), I'd like a new module that fixes an input to a particular value.

The input would no longer be shown to the user in the form.

In the descriptor, the spec would look like:

```json
"custom": {
  "cbrain:integrator_modules": {
    "InputValueFixer": {
      "inputid1": "somevalue",
      "inputid2": 2345
    }
  }
}
```

**Comment by prioux** [comment]
The PR that will implement this will have only 2 new files (one of them a symlink) and should consist in about 20 lines of code, total (rough estimate).

**Comment by MontrealSergiy** [comment]
@prioux what about the input-value dependency (disables and requires and such), group validation , should those be adjusted (might need more than 20 lines of code) or dropped?

**Comment by prioux** [comment]
No, don't make any adjustments. We only want to bare functionality I describe here.

We expect a programmer who use the module will do it responsibly and not choose an input that has other interfering rules applied to it.
